Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rasa/rasa-sdk:1.8.1
+FROM rasa/rasa-sdk:1.9.0
```

Comment:
Neesd to be rasa[spacy]~=1.9.1

Reply:
In the requirements though right? Not the dockerfile

Reply:
yes! just wanted to comment it somewhere before you merged, since I had accepted

Reply:
sure, I changed it, I don't think you approved though?